stdlib/fetch.py

```diff
@@ -79,10 +80,15 @@ def fetch_file(file: str):
         file,
     )
 
+    new_name = rename
```

Comment:
```suggestion
    new_name = rename or os.path.basename(file)
```
And then you can get rid of the `if` below

Reply:
swag